examples/cephfs/snapshot.yaml

```diff
@@ -1,5 +1,10 @@
 ---
-apiVersion: snapshot.storage.k8s.io/v1beta1
+# Snapshot API version compatibility matrix:
+# v1.17 =< k8s < v1.20, use v1betav1
+# k8s >= v1.20, use v1
+# v2.x =< snapshot-controller < v4.x, use v1beta1
+# snapshot-controller >= v4.x, use v1
```

Comment:
just mention that sidecar and controller should be of the same version as much as possible

Reply:
Sure :+1: 

Thanks Madhu!